backend/service/github/github.go

```diff
@@ -345,20 +344,15 @@ func (s *svc) GetFile(ctx context.Context, ref *RemoteRef, path string) (*File,
 	return f, nil
 }
 
-func (s *svc) CompareCommits(ctx context.Context, ref *RemoteRef, compareSHA string) (*scgithubv1.CommitComparison, error) {
+/*
+ * Rather than calling GetCommit() multiple times, we can use CompareCommits to get a range of commits
+ */
+func (s *svc) CompareCommits(ctx context.Context, ref *RemoteRef, compareSHA string) (*githubv3.CommitsComparison, error) {
```

Comment:
The only concern I have about changing the return type is that it'll break something but I do think this type makes more sense

Reply:
Another option is to just go back to the way I had it before (with a separate function) and just leave a TODO to maybe combine the 2 functions.  I thought about adding to the proto as well - the reason why I didn't do that was I didn't see a good way to import all the fields from the github structs in proto form.